src/util/misc.ts

```diff
@@ -334,3 +335,27 @@ export function stripHttpFromUrl(url: string): string {
   return url.replace(FIND_HTTP_REGEX, '');
 }
 
+
+export function isImageUrl(url: string): boolean {
+  return IMAGE_URL_REGEX.test(url);
+}
+
+export function isWebpCandidateImageUrl(url: string): boolean {
+  return WEBP_CANDIDATE_IMAGE_FILENAME_REGEX.test(url);
+}
+
+export function isImageMimeType(mimeType: string): boolean {
+  return IMAGE_MIME_REGEX.test(mimeType);
+}
+
+export function isBitmapImageMimeType(mimeType: string): boolean {
+  return BITMAP_IMAGE_MIME_REGEX.test(mimeType);
+}
+
+export function shouldConvertImageFilenameToWebp(url: string, webp: boolean) {
+  return webp && WEBP_CANDIDATE_IMAGE_FILENAME_REGEX.test(url);
+}
+
+export function isWebpCandidateImage(webp: boolean, cssDependency: boolean, content_type: string){
+  return (webp && !cssDependency && content_type !== 'image/gif')
+}
```

Comment:
* Name should be `isWebpCandidateImageMimeType`
* This shoudl use a revex
* Not handling of `cssDependency` at this level

Reply:
@kelson42 should I also remove webp argument from here?

Reply:
actually if we are using 2 profiles then there is no need at all for this function

Reply:
Probably, indeed, a bit unsure